packages/autorest.python/autorest/codegen/models/__init__.py

```diff
@@ -159,7 +159,7 @@ def build_type(yaml_data: Dict[str, Any], code_model: CodeModel) -> BaseType:
         response.fill_instance_from_yaml(yaml_data, code_model)
     else:
         object_type = yaml_data.get("type")
-        if object_type is None:
+        if object_type not in TYPE_TO_OBJECT:
```

Comment:
@iscai-msft modelerfour output format value as type value so we can't distinguish string type with unrecognized format or just unrecognized type. Now that both scenario cause generation failure, I think we could handle unrecognized type as string uniformly. What do you think about it?

Reply:
Yeah that sounds good to me, thanks!

Reply:
@iscai-msft I have resolved the conflict. Please help merge and release a new version. Thanks!